doc/ref_cert/lfn/chapters/chapter05.md

```diff
@@ -146,37 +189,77 @@ across multiple tenant’s environments.
 
-Allows executing the test cases from multiple 3rd party test tools and producing
-the results
+In any telco environment, heterogeneous is inevitable and need for executing the
+test cases from different test tools is required. so system would support to
+on-board the 3rd party test tools and corresponding test cases from them and
+allows executing the test cases from those 3rd party test tools.
 
 #### 5.3.10 Certification management & integration
 
 Platform should have integration with OVP certification portal for submitting
-results with OVP defined format
-
-#### 5.3.11 OVP portal integration
+results with OVP defined format.
 
-#### 5.3.12 User & System interfaces
+It should enable repository of certified VNFs which can be used for testing
+validation and performance.
 
-User interface:
+#### 5.3.11 User & System interfaces
 
->   CLI
+**User interface**:
 
->   Web portal
+*   CLI
+*   Web portal
 
-Programming interface: REST API
+**Programming interface**:
 
-#### 5.3.13 Deliverables
+* REST API
+* gRPC
 
-Docker image based installation
+#### 5.3.12 Deliverables
+Platform should be able to get deployed in both container and cloud environments.
+so following model deliverables would enable it:
 
-Standalone installation scripts and zip artifact
+* Docker image based installation
+* Standalone installation scripts and zip artifact
 
 <a name="5.4"></a>
-### 5.4 Certification Process
+### 5.4 Certification Methodology
+Perform the required VNF certtification based on scope defined in this chapter and by
+leveraging the test cases and tools which provides the features defined under
+platform requirements as above.
 
-This process details the end-end steps required for certifying the given VNF by
-using ths platform (providing above requirements), producing the test results to
+It defines the end-end Certification Process required for certifying the given VNF by
+using this platform (providing above requirements), producing the test results to
 certification portal , review and granting the badge to this VNF.
 
+[More details](https://wiki.lfnetworking.org/display/LN/VNF+Certification+Lifecycle)
+
 #### 5.4.1 Entry & Exit criteria
+
+**Entry criteria**: Before entering into the VNF certification, VNF needs to
+satisfy the following requirements as entry pass:
+
+* *Environment Requirements* : Details on required dependent system which are
+required to deploy and test VNF.
+* *VNF artifact* : VNF cloud (native) image, VNF configurations and guidelines,
+etc
+* *NFVi profiles*: List of supporting OVP Certified CNTT compliant NFVi
+* Completed Security review report
+* Vendor specific test cases and its deployment and usage guidelines
+
+**Exit criteria**: VNF certification testing should completed with following
+exit criteria:
+
+* All required test cases should be passed
+* No outstanding high severity issues and other know issues to be documented
+* Release notes
+* Provided with required installation guide, configuration guide, etc.

```

Comment:
I didn't get the installation guide, configuration guide as a requirement for exit criteria for vnf testing. Can you elaborate ?

Reply:
@msidana these guidelines would help while opeartor try the vnf in their environment and could include any MANO specific guidelines as well as part of it.

Reply:
@msidana - please review to see if we can close

Reply:
1) How would it help if its a part of **exit criteria** ? 
2) How would the presence/absence of these guides help in certification. These shouldn't fall in the purview of certification.

Reply:
@msidana , from an Operator perspective, this is an expected artifact before we take receipt of a VNF/CNF.  When a badge is provided from OPNFV/LFN/GSMA, not only is evidence required demonstrating infrastructure verification and test validation with results, but documentation for release notes, and installation guides for implementing the VNF/CNF.    This will help speed up acceptance and adoption of VNFs/CNFs by Operators.

Any other questions, or can we close?

Reply:
Yes, I agree with your point. But then it makes more sense to keep it as a part of **entry criteria** rather than exit criteria. These artefacts should be an entry barrier.